searx/data/engine_traits.json

```diff
@@ -4146,6 +4619,7 @@
         "km",
         "kn",
         "ko",
+        "koi",
         "krc",
```

Comment:
Its not a part of this PR

Reply:
How can I only generate traits for a specific engine? I've used `make data.traits`, but it does a bunch of changes I don't want it to.

Reply:
Its not possible :) .. I add only those hunks to the git commit I want in the commit and revert the rest .. Its not a problem, I can do it in my review if you want.

Reply:
Should be fixed now.